star-rx72n-firmware/Dockerfile

```
@@ -4,11 +4,23 @@ FROM msalau/rx-elf-gcc:latest
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    cmake \
+    ca-certificates \
+    curl \
     make \
     git \
     && rm -rf /var/lib/apt/lists/*
 
+# Install modern CMake (>= 3.20)
+ARG CMAKE_VERSION=3.27.9
+RUN curl -fsSL "https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz" \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)